ShijiLighting/GHIElectronics.TinyCLR.Drivers.ShijiLighting.APA102C/GHIElectronics.TinyCLR.Drivers.ShijiLighting.APA102C.nuspec

```diff
@@ -0,0 +1,23 @@
+<?xml version="1.0" encoding="utf-8"?>
+
+<package xmlns="http://schemas.microsoft.com/packaging/2011/08/nuspec.xsd">
+  <metadata>
+    <id>GHIElectronics.TinyCLR.Drivers.ShijiLighting.APA102C</id>
+    <version>1.0.0-preview1</version>
+    <authors>GHI Electronics, LLC</authors>
+    <owners>GHI Electronics, LLC</owners>
+    <projectUrl>https://ghielectronics.com/</projectUrl>
+    <iconUrl>http://files.ghielectronics.com/downloads/Logos/GHIElectronicsTinyCLRNuGet.png</iconUrl>
+    <licenseUrl>http://files.ghielectronics.com/downloads/TinyCLR/License.txt</licenseUrl>
+    <requireLicenseAcceptance>true</requireLicenseAcceptance>
+    <description>ShijiLighting APA102C driver for TinyCLR OS.</description>
+    <copyright>Copyright GHI Electronics, LLC 2018</copyright>
+    <tags>TinyCLR TinyCLROS APA102C</tags>
+    <dependencies>
+      <dependency id="GHIElectronics.TinyCLR.Core" version="1.0.0-preview1" />
```

Comment:
You should add SPI and GPIO as a dependency. The general rule is whatever top-level nuget packages your project references you be added here.